src/os/ui/providerimport.js

```diff
@@ -291,3 +291,11 @@ os.ui.ProviderImportCtrl.prototype.formDiff = function() {};
  * @protected
  */
 os.ui.ProviderImportCtrl.prototype.getConfig = function() {};
+
+
+/**
+ * Handles URL changes
+ *
+ * @export
+ */
+os.ui.ProviderImportCtrl.prototype.validateUrl = function() {};
```

Comment:
Sorry I suggested the wrong place for this. I think it makes more sense on the `SingleUrlProviderImportCtrl` class. Could you move it there please?